cli/fathom_web/test/test_extract.py

```diff
@@ -0,0 +1,32 @@
+from ..commands.extract import BASE64_DATA_PATTERN
+
+
+def test_common_example():
+    """Confirm we handle to well behaving case"""
```

Comment:
to→the

And it's a sentence, so period! :-) If you want to get fancy, you can hyphenate "well-behaving". :-o

Reply:
Whoops. Thanks for catching that!